scripts/svm/queryFills.ts

```diff
@@ -47,27 +46,30 @@ async function queryFills(): Promise<void> {
       console.log("No fill events found for the given seed.");
       return;
     }
+
     console.log("Fill events fetched successfully:");
     fillEvents.forEach((event, index) => {
       console.log(`Fill Event ${index + 1}:`);
       console.table([
-        { Property: "inputToken", Value: strPublicKey(event.data.inputToken) },
-        { Property: "outputToken", Value: strPublicKey(event.data.outputToken) },
+        { Property: "inputToken", Value: new PublicKey(event.data.inputToken).toString() },
+        { Property: "outputToken", Value: new PublicKey(event.data.outputToken).toString() },
         { Property: "inputAmount", Value: event.data.inputAmount.toString() },
         { Property: "outputAmount", Value: event.data.outputAmount.toString() },
         { Property: "repaymentChainId", Value: event.data.repaymentChainId.toString() },
         { Property: "originChainId", Value: event.data.originChainId.toString() },
         { Property: "depositId", Value: event.data.depositId.toString() },
-        { Property: "depositIdNum", Value: u8Array32ToInt(event.data.depositId).toString() },
         { Property: "fillDeadline", Value: event.data.fillDeadline.toString() },
         { Property: "exclusivityDeadline", Value: event.data.exclusivityDeadline.toString() },
-        { Property: "exclusiveRelayer", Value: strPublicKey(event.data.exclusiveRelayer) },
-        { Property: "relayer", Value: strPublicKey(event.data.relayer) },
-        { Property: "depositor", Value: strPublicKey(event.data.depositor) },
-        { Property: "recipient", Value: strPublicKey(event.data.recipient) },
-        { Property: "messageHash", Value: event.data.messageHash.toString() },
-        { Property: "updatedRecipient", Value: strPublicKey(event.data.relayExecutionInfo.updatedRecipient) },
-        { Property: "updatedMessageHash", Value: event.data.relayExecutionInfo.updatedMessageHash.toString() },
+        { Property: "exclusiveRelayer", Value: new PublicKey(event.data.exclusiveRelayer).toString() },
+        { Property: "relayer", Value: new PublicKey(event.data.relayer).toString() },
+        { Property: "depositor", Value: new PublicKey(event.data.depositor).toString() },
+        { Property: "recipient", Value: new PublicKey(event.data.recipient).toString() },
+        { Property: "message", Value: event.data.message.toString() },
```

Comment:
fill events don't have full message anymore. instead its messageHash

Reply:
ye, this will be broken now if used. I think it could be due to a merge issue you had.